packages/twenty-server/src/workspace/workspace-query-runner/workspace-query-runner.service.ts

```diff
@@ -306,4 +362,26 @@ export class WorkspaceQueryRunnerService {
 
     return this.parseResult(result, targetTableName, command);
   }
+
+  async addJobsToQueue<JobsData>(
```

Comment:
addJobsToQueue seems too generic. triggerWebhooks?
I also think that the templating with Record[] | undefined is not useful as we are always using it with `Record[] | undefined`